apps/playground/src/app/components/help-inline/help-inline.module.ts

```diff
@@ -1,7 +1,7 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
-import { SkyHelpInlineModule } from '@skyux/help-inline';
+import { SkyHelpInlineModule } from '@skyux/indicators';
```

Comment:
Could you make a new route for `help-inline-legacy` that uses the deprecated help inline? I think we wanted this route to use the new one.